catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -285,6 +310,11 @@ public CswEndpoint(BundleContext context, CatalogFramework ddf, FilterBuilder fi
             TransformerManager inputManager) {
         this(context, ddf, filterBuilder, manager, schemaManager, inputManager);
         this.uri = uri;
+        try {
```

Comment:
Instead of a separate ctor, do you think we should just have a package-private setter for the URI? If you decide to keep the ctor, can we make it package-private instead of `public`?